### DISCUSSION TITLE
멀티룸 씬전환

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#389] [5. Q&A]
캐릭터가 특정 장소에 갔을 때 씬전환이 되도록 하고 싶은데, 그 장소에 있지 않은 다른 캐릭터도 동시에 씬전환이 됩니다. 

개별적으로 씬 전환할 수 있는 방법이 있나요?

**Comment by moondory77-naverz** [comment]
안녕하세요. 
멀티플레이 컨텐츠에서, ZEPETO.Character.Controller 모듈을 통해 생성되는 제페토 캐릭터 인스턴스는
기본적으로 씬에 종속된 인스턴스로, 씬 전환 시 유지되지 않습니다.

씬 전환에 대해, 컨텐츠 레벨에서 캐릭터 인스턴스를 유지하거나
혹은 제거 후 재생성 하는 등의 로직 처리가 필요합니다.

도움이 될 수 있다면 좋겠습니다.
감사합니다.
